Scripts/test-CocoaPods-integration.sh

```diff
@@ -41,41 +41,43 @@ rm -rf $PROJECT_NAME
 
 mkdir -p $PROJECT_NAME && cd $PROJECT_NAME
 
-# Create a new Xcode project.
-swift package init
-
-# Create the Package.swift.
-echo "// swift-tools-version:5.3
-// The swift-tools-version declares the minimum version of Swift required to build this package.
-
-import PackageDescription
-
-let package = Package(
-    name: \"TempProject\",
-    platforms: [
-        .iOS(.v12)
-    ],
-    products: [
-        .library(
-            name: \"TempProject\",
-            targets: [\"TempProject\"]),
-    ],
-    dependencies: [],
-    targets: [
-        .target(
-            name: \"TempProject\",
-            dependencies: []),
-        .testTarget(
-            name: \"TempProjectTests\",
-            dependencies: [\"TempProject\"]),
-    ]
-)
-
-" > Package.swift
-
-swift package update
-
-swift package generate-xcodeproj
+echo "
```

Comment:
This file change is substantial and is not very clear, can you provide brief explanation what's happening in here?